### DISCUSSION TITLE
[A-L] (13 Febbraio 2024) Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#408] [Esami A-L]
### Fila A
Dimostrare per Induzione che, per ogni $\Large n \geq 1$,

$$
    \Large \sum_{i=1}^{n} \frac{1}{i(i+1)} = \frac{n}{n+1}.
$$

### Fila B

Dimostrare per Induzione che, per ogni $\Large n \geq 0,\ n(n + 1)$ è pari.

### Entrambe le file

Indicare con precisione:

- il Caso Base (1.5 punti),

- l’Ipotesi Induttiva (1.5 punti)

- e la dimostrazione del Passo Induttivo (2 punti).

**Comment by DavideGalilei** [comment]
### 🅰️ Fila A
Dimostrare per induzione che, per ogni $\Large{n \geq 1}$,

$$\Large{\sum_{i=1}^{n} \frac{1}{n(n + 1)} = \frac{n}{n + 1}}$$

#### Caso Base
$\Large{P(1): \sum_{i=1}^{n} \frac{1}{n(n + 1)} = \frac{1}{1 + 1} = \frac{1}{2}}$

#### Ipotesi Induttiva
$\Large{P(n): \sum_{i=1}^{n} \frac{1}{n(n + 1)} = \frac{n}{n + 1}}$

#### Passo Induttivo
$\Large{P(n+1): \sum_{i=1}^{n+1} \frac{1}{n(n + 1)} = (\sum_{i=1}^{n} \frac{1}{n(n + 1)}) + \frac{1}{(n+1)(n+2)} =}$
$\Large{\frac{n}{n + 1} + \frac{1}{(n+1)(n+2)} = \frac{n(n+2) + 1}{(n+1)(n+2)} = \frac{n^2 + 2n + 1}{(n+1)(n+2)} = \frac{(n+1)(n+1)}{(n+1)(n+2)} = \frac{n+1}{n+2} = \frac{n+1}{(n+1) + 1}}$

### 🅱️ Fila B
Dimostrare per Induzione che, per ogni $\Large{n \geq 1}$, $\Large{n(n+1)}$ è pari.

#### Caso Base
$P(1): 1(1+1) = 2$
Il numero è pari ✅

#### Ipotesi Induttiva

$$\Large{P(n) = n(n+1)}$$ è pari. Dimostrazione (non per induzione) nel caso $\Large{n}$ fosse pari e dispari:

$$\Large{P(n) = \begin{cases} 
n(n+1) = 2k(2k+1) = 4k^2 + 2k = 2(2k^2 + k) & \text{if } n = 2k \text{ (pari)} \\
n(n+1) = (2k+1)(2k+1+1) = 4k^2 + 6k + 2 = 2(2k^2 + 3k + 1) & \text{if } n = 2k + 1 \text{ (dispari)}
\end{cases}}$$

P(n) è pari ✅

#### Passo induttivo
$$\Large{P(n+1) = (n+1)(n+1 + 1) = (n+1)(n+2) = n^2 + 3n + 2 = (n^2 + n) + (2n + 2) = n(n + 1) + 2(n + 1)}$$

Ci siamo ricondotti così all'ipotesi induttiva $\Large{n(n + 1)}$ che sapevamo essere pari, sommandoci $\Large{2(n + 1)}$ che a sua volta è un numero pari, e pari+pari=pari ($\large{2k+2k = 4k = 2(2k)}$)
